object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -279,6 +304,253 @@ private static String getExactStringValue(Tree tree, ValueAnnotatedTypeFactory f
     return null;
   }
 
+  @Override
+  public void postAnalyze(ControlFlowGraph cfg) {
+    checkAlwaysCall(cfg);
+    super.postAnalyze(cfg);
+  }
+
+  private void checkAlwaysCall(ControlFlowGraph cfg) {
+
+    Set<Pair<BlockImpl, Set<Tree>>> visited = new HashSet<>();
+    visited.add(Pair.of((BlockImpl) cfg.getEntryBlock(), Collections.emptySet()));
```

Comment:
For clarity let's store `Pair.of((BlockImpl) cfg.getEntryBlock(), Collections.emptySet())` in a local variable and re-use it for `visited` and `worklist` initialization